docs/hotspot-makers/maker-apps.mdx

```diff
@@ -18,8 +18,8 @@ Followed by a listing of the available Maker Apps.
 
 Beginning March 2022, all [HIP-19](https://github.com/dewi-alliance/hotspot-manufacturers#readme)
 Hotspot Manufacturers (aka Makers) are required to support their customers with a bespoke Mobile App
-to onboard their Hotspots, maintain their Hotspots with diagnostic tools, and various basic
-troubleshooting tools.
+to onboard their Hotspots, assert Hotspot locations and maintain their Hotspots with diagnostic tools, 
```

Comment:
Should be a comma after 'Hotspot locations' (Oxford comma).
One too many 'and's in this sentence:
"assert Hotspot locations **and** maintain"
"diagnostic tools, **and** various basic troubleshooting tools."